zookeeper-metrics-providers/zookeeper-prometheus-metrics/src/main/java/org/apache/zookeeper/metrics/prometheus/PrometheusMetricsProvider.java

```diff
@@ -607,4 +607,20 @@ private void reportMetrics(final Runnable task) {
             task.run();
         }
     }
+    // make it worse
+    public void dummy() {
+        if (true) {
+            if (true) {
+                if (true) {
+                    if (true) {
```

Comment:
Manual Comment....

### ❌ New issue: Deep, Nested Complexity Impact on Code Health Score -0.54 (-0.54)
dummy has a nested complexity depth of 4, threshold = 4
### Why does this problem occur?
Measures the maximum nesting depth inside each function.